clients/web/src/views/widgets/MetadataWidget.js

```diff
@@ -317,7 +317,16 @@ var JsonMetadatumEditWidget = MetadatumEditWidget.extend({
         this.editor = new JSONEditor(this.$el.find('.g-json-editor')[0], {
             mode: 'tree',
             modes: ['code', 'tree'],
-            onError: function () {
+            onModeChange: () => {
+                // JSONEditor removes tooltip-containing elements without destroying their
+                // Bootstrap-styled tooltips first. Since this callback is not called until after
+                // the original tooltip-containing element is destroyed, the only way to clean up is
+                // to remove the tooltip from DOM.
+                // When used in an external app with no tooltip styling enabled, this call should
+                // have no effect (assuming there's no other ".tooltip" elements.
+                $('.tooltip').remove();
```

Comment:
Should this be `this.$`?

Reply:
Unfortunately, no. The tooltip is attached directly below `#g-app-body-container`, which is much higher in the hierarchy than this widget's element. I was considering using the selector `'#g-app-body-container>.tooltip'), but I wanted to keep it general enough to also correct tooltip problems if this was reused in another external app.